rest/app.py

```diff
@@ -1,78 +1,75 @@
 from flask import Flask, render_template
-from os import path
 import json
 import config
 
 from flask import request
 
-app = Flask(__name__)
-
+from sql.tasks import SQLTasks
+from sql.users import SQLUsers
 
-data_dir = path.join(path.dirname(__file__), 'data')
-users_data = path.join(data_dir, 'users.json')
-tasks_data = path.join(data_dir, 'tasks.json')
-
-with open(users_data, 'r') as f:
-    users = json.loads(f.read())
+app = Flask(__name__)
 
-with open(tasks_data, 'r') as f:
-    tasks = json.loads(f.read())
 
 config = config.RestConfig()
+sql_users = SQLUsers()
+sql_tasks = SQLTasks()
 
 
 @app.route('/')
 @app.route('/index')
 def index():
-    return render_template('index.html', users=users, tasks=tasks)
+    return render_template('index.html')
 
 
 @app.route('/users', methods=['GET'])
 def get_users():
-    return json.dumps(users)
+    return json.dumps(sql_users.get_all_users())
 
 
 @app.route('/users', methods=['POST'])
 def add_user():
     user = json.loads(request.json)
-    users.append(user)
-    return render_template('index.html'), 201
+    user_name = user['name']
+    added_user = sql_users.add_user(user_name)
+    return json.dumps(added_user), 201
 
 
 @app.route('/users/<int:user_id>', methods=['DELETE'])
 def delete_user(user_id):
-    for user in users:
-        if user['id'] == user_id:
-            users.remove(user)
-            return render_template('index.html'), 201
-    return not_found(404)
+    sql_users.delete_user(user_id)
+    return render_template('index.html'), 204
 
 
 @app.route('/users/<int:user_id>', methods=['GET'])
 def get_user(user_id):
-    for user in users:
-        if user['id'] == user_id:
-            return json.dumps(user)
+    user = sql_users.get_user(user_id)
+    if user:
+        return json.dumps(user)
     return not_found(404)
 
 
 @app.route('/users/<int:user_id>/tasks', methods=['GET'])
-def get_user_tasks(user_id, start='', end=''):
-    # ToDo(stasya): add request to database
-    user_tasks = list()
-    for task in tasks:
-        if task['user_id'] == user_id:
-            # SQL request for the given period
-            user_tasks.append(task)
-
-    return json.dumps(user_tasks)
+def get_user_tasks(user_id):
+    period = request.args
+    month = None
+    day = None
+    if period:
+        year = period['year']
+        if 'month' in period:

```

Comment:
You can use the construction like:
_month = period.get('month', None)_
It lets you skip setting value before and _if month in ..._